orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/ContainerOrchestrator.java

```diff
@@ -13,6 +14,7 @@ void startAmbariServer(ContainerOrchestratorCluster cluster, String dbImageName,
     void startConsulWatches(ContainerOrchestratorCluster cluster, String imageName, int count) throws CloudbreakOrchestratorException;
 
     boolean areAllNodesAvailable(String gatewayAddress, Set<Node> nodes);
+    List<String> getAllAvailableNode(String gatewayAddress, Set<Node> privateAddressesOfNodes);
```

Comment:
please rename the method to getAvailableNodes and privateAddressesOfNodes to nodes